eZ/Publish/API/Repository/Tests/TrashServiceTest.php

```diff
@@ -10,6 +10,7 @@
 
 use eZ\Publish\API\Repository\Repository;
 use eZ\Publish\API\Repository\URLAliasService;
+use eZ\Publish\API\Repository\Values\User\User;
```

Comment:
unsure why ezrobot did not complain about this one, does to seem to be used?

Reply:
True, that's a relic from an earlier iteration of the test.

Reply:
@andrerom Double checking this in https://github.com/ezsystems/ezpublish-kernel/pull/2149